src/externals/pio2/src/clib/ncputgetparser.pl

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/perl 
```

Comment:
Same here, ncputgetparser.pl was also deleted (Sep 17, 2016) from pio2 master (https://github.com/PARALLELIO/ParallelIO.git) and is no longer needed